### DISCUSSION TITLE
Inline Katex và Image

### DISCUSSION THREAD

**Discussion by TungNguyen-C4U** [hunghg255/reactjs-tiptap-editor#78] [Ideas]
Hi @hunghg255 @condorheroblog 

Thư viện của mấy bạn đã giúp ích rất nhiều cho dự án của mình. Tuy nhiên khi mình muốn thêm công thức Toán hoặc hình ảnh trên cùng một dòng thì Editor mình chưa có, không biết bên bạn có roadmap để phát triển tính năng này không nhỉ?

Cám ơn mọi người

**Comment by hunghg255** [comment]
> <img alt="" width="32" height="32" src="https://avatars.githubusercontent.com/u/26731234?s=64&amp;v=4">
> TungNguyen-C4U

mình cũng không có roadmap hay gì đâu á
cái inline này bạn có thể request ở phân issue nhé, mình sẽ nghiên cứu làm, hoặc có ai đó hứng thú open source họ làm thì tốt
